cms/djangoapps/contentstore/rest_api/v1/views/tests/test_assets.py

```diff
@@ -0,0 +1,281 @@
+
+
+"""
+Tests for the xblock view of the Studio Content API. This tests only the view itself,
+not the underlying Xblock service.
+It checks that the assets_handler method of the Xblock service is called with the expected parameters.
+"""
+from unittest.mock import patch
+from django.http import JsonResponse
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+from cms.djangoapps.contentstore.tests.test_utils import AuthorizeStaffTestCase
+
+
+ASSET_KEY_STRING = "asset-v1:dede+aba+weagi+type@asset+block@_0e37192a-42c4-441e-a3e1-8e40ec304e2e.jpg"
+
+
+class AssetsViewTestCase(AuthorizeStaffTestCase):
```

Comment:
Have a look at the AuthorizeStaffTestCase in cms/djangoapps/contentstore/tests/test_utils.py! This is a class I'm quite happy about that I added in the previous PR on xblocks. Basically there is a test for authorization of course staff only that runs the `make_request` method of the child class and makes sure there is 403 if the user is not a course author for the course in question.

Here is the test that checks that course authors for other courses cannot access the API, for example:
```python
    def test_instructor_in_another_course(self, expect_status=status.HTTP_403_FORBIDDEN):
        self.client.login(
            username=self.other_course_instructor.username,
            password=self.password
        )
        response = self.make_request()
        assert response.status_code == expect_status
```

So every class that inherits from this will have the authorization tests included.

Reply:
Excellent 👍🏻